### PR TITLE
Reduce linter problems

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
I have reduced the number of linter problems from 215 to 8. I have had almost all of them fixed automatically. The others were minor, so I don't expect any impact on functionality 🙂

The remaining 8 problems would need a closer look.

### CODE REVIEW

File: MMM-PublicTransportBerlin.js [hunk 1]
```diff
@@ -1,37 +1,62 @@
...
-    fadeUnreachableDepartures: true,    // Should unreachable departures be faded away from the reachable departures line?
-    fadeReachableDepartures: true,      // Should reachable departures be faded away from the reachable departures line?
-    fadePointForReachableDepartures: 0.25, // The point to start fading the reachable departures
-    excludeDelayFromTimeLabel: false,   // Should the delay time be excluded from the time label?
-    showDirection: true,                // Adds direction of the module instance to the header if the instance is directed
-    animationSpeed: 3000,               // Speed of the update animation. (Milliseconds)
-    timezone: "Europe/Berlin",          // Timezone
-    shortenStationNames: true           // Shorten station names? See https://github.com/derhuerst/vbb-short-station-name
+    // The name of this module
+    name: "MMM-PublicTransportBerlin",
```
Comment (rejas) [inline MMM-PublicTransportBerlin.js:7]: the old formatting looks better IMO but IANTM (i am not the maintainer ;-)

Comment (KristjanESPERANTO) [reply MMM-PublicTransportBerlin.js:7]: Would there be a more elegant way to keep the old formatting without disabling prettier for the whole file?

Comment (KristjanESPERANTO) [reply MMM-PublicTransportBerlin.js]: I undid the changes in the default section so we can move forward with this PR.

**Review by rejas — APPROVED** [review]

**Comment by deg0nz** [comment]
Hey you guys, thanks again for your effort!

It is forseeable, that I won't have much time for maintenance until at least august. Should/may I add one or both of you as maintainers for this repo? 

Since you both have done a lot of stuff here, I trust you to behave wisely and in my interests :)

**Comment by rejas** [comment]
I can take that responsibility :-) I promise I will only maintain and not feature-bust it :-D

**Comment by KristjanESPERANTO** [comment]
I would be open too, but I'm sure it's enough for now if Rejas becomes an additional maintainer 🙂

**Comment by KristjanESPERANTO** [comment]
Thanks for the trust to make me a maintainer too, @deg0nz! 🙂

**Comment by rejas** [comment]
Just a quick question: Does anyone else get this error when using node > v14 ?

Error: Cannot find module '/home/veeck/Projekte/MMM-PublicTransportBerlin/node_modules/json-schema-traverse/index.js'. Please verify that the package.json has a valid "main" entry

**Comment by rejas** [comment]
Nevermind, a clean reinstall fixed it...

**Pull request merged**